requirements.txt

```diff
@@ -57,6 +62,7 @@ smmap==0.9.0
 sockjs-tornado==1.0.1
 sourcemap==0.1.8
 tornado==2.4.1
+twilio==5.2.0
 python-twitter==1.1
 ujson==1.33
 uritemplate==0.6
```

Comment:
@sharmaeklavya2 do you want to check out the Python 2 compatibility story for these packages?